api/clients/disperser_client_v2.go

```diff
@@ -109,15 +121,11 @@ func (c *disperserClientV2) DisperseBlob(
 		return nil, [32]byte{}, api.NewErrorInternal("uninitialized signer for authenticated dispersal")
 	}
 
```

Comment:
check if `c.accountant` is nil? 
we can instruct how to initialize accountant in the error message